### ISSUE TITLE
Should this repo be included in Palm?

### ISSUE THREAD

**Issue by mtyaka** [openedx/edx-enterprise-subsidy-client#11]
The Build-Test-Release Working Group is preparing to cut branches for Palm, the next Open edX release. The cut will happen on April 11th.

The [openedx.yml](https://github.com/openedx/edx-enterprise-subsidy-client/blob/8240afa093994c370f6c6054049c71e8ed210d3b/openedx.yaml#L22) currently says that this repo should "maybe" be included in the next release.

Is the repo ready to be included in Palm? Or should we wait until Quince, scheduled to be released in December?

@iloveagent57

**Comment by iloveagent57** [comment]
@mtyaka nope, we should wait for Quince, this one won't quite be ready for Palm

**Comment by nedbat** [comment]
@iloveagent57 Can you update the openedx.yaml file to indicate this?
